deepseg.cc

```diff
@@ -377,6 +398,8 @@ int main(int argc, char* argv[]) {
 	printf("Startup: %ldns\n", diffnanosecs(ti.lastns,ti.bootns));
 
 	bool filterActive = true;
+	calcinfo_t calcinfo = { modelname, threads, width, height, debug };
+	init_tensorflow(calcinfo);
```

Comment:
These two lines don't compile in the context of this commit.

Reply:
fixed